cli/ceph/orch/label.py

```diff
@@ -0,0 +1,27 @@
+from cli import Cli
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class Label(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(Label, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} host label"
+
+    def add(self, **kw):
+        """
+        Method add _no_schedule label in the node
+        Args:
+            node: cluster node
+            label: label name
+        """
+
+        node = kw["node"]
```

Comment:
Same as below

Reply:
Updated

Reply:
Here method signature should be - 
```
def add(self,  node, label):
```
As `node` & `label` are mandatory parameter. Use `**kw` in case command supports any optional parameter. For example `--force`.

Reply:
Updated